src/main/java/properties/EnvironmentConfig.java

```diff
@@ -0,0 +1,84 @@
+package properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+/***
+ * Project environment data
+ */
+public class EnvironmentConfig {
+
+    private static final String MAIN_URL = "url";
+    private static final String REGISTRATION_URL = "regURl";
+    private static final String CATALOGUE_URL = "catalogueURL";
+    private static final String CART_URL = "cartURL";
+    private static final String PASSWORD = "password";
+    private static final String LOGIN = "login";
+
+    private final static String ENVIRONMENT_PROPERTIES_PATH = "src/main/resources/application-%s.properties";
+    private final static Properties properties;
+
+    @Getter
+    @AllArgsConstructor
+    enum Environment {
+        QA("qa"),
+        DEV("dev"),
+        LOCAL("local");
+
+        String name;
+
+        public static Environment setEnvironment() {
```

Comment:
 I think it makes sense to rename this method to getEnvironment() as you are not setting. anything there

Reply:
fixed